_integration-schemas/ebay/v1/orders.md

```diff
@@ -7,7 +7,7 @@ name: "orders"
 doc-link: "https://developer.ebay.com/api-docs/sell/fulfillment/resources/order/methods/getOrders"
 singer-schema: "https://github.com/singer-io/tap-ebay/blob/master/tap_ebay/schemas/orders.json"
 description: |
-  The {{ table.name }} table contains details about all orders in your {{ integration.display_name }} seller account.
+  The {{ table.name }} table contains details about all orders in your {{ integration.display_name }} seller account. This table must use a request with a Start Date set that is within 90 days.
```

Comment:
What error would the user receive if they set it beyond 90 days? Include that here so they can identify the problem when they receive the error.